static/app/views/performance/newTraceDetails/traceDrawer/details/noData.tsx

```diff
@@ -5,24 +5,15 @@ import {Button} from 'sentry/components/button';
 import useFeedbackWidget from 'sentry/components/feedback/widget/useFeedbackWidget';
 import {IconGroup} from 'sentry/icons';
 import {t, tct} from 'sentry/locale';
-import type {Organization} from 'sentry/types';
 import {TraceDrawerComponents} from 'sentry/views/performance/newTraceDetails/traceDrawer/details/styles';
+import type {TraceTreeNodeDetailsProps} from 'sentry/views/performance/newTraceDetails/traceDrawer/tabs/traceTreeNodeDetails';
 import {
   makeTraceNodeBarColor,
   type NoDataNode,
-  type TraceTree,
-  type TraceTreeNode,
 } from 'sentry/views/performance/newTraceDetails/traceTree';
 import {Row} from 'sentry/views/performance/traceDetails/styles';
 
-interface NoDataDetailsProps {
-  node: NoDataNode;
-  onParentClick: (node: TraceTreeNode<TraceTree.NodeValue>) => void;
-  organization: Organization;
-  scrollToNode: (node: TraceTreeNode<TraceTree.NodeValue>) => void;
-}
-
-export function NoDataDetails(props: NoDataDetailsProps) {
+export function NoDataDetails(props: TraceTreeNodeDetailsProps<NoDataNode>) {
```

Comment:
These were tedious to always update across the different render functions, so I consolidated them into a single prop type with the generic node